lib/cylc/gui/updater_tree.py

```diff
@@ -179,14 +184,18 @@ def update_gui(self):
 
         """
         self.action_required = False
+        if self.state_summary:
```

Comment:
When I start `cylc gui` in this mode, I get:
```
AttributeError: 'TreeUpdater' object has no attribute 'state_summary'
```
coming from this line. Should we set it as empty further up?